src/capi-multiple-paidfor/test.json

```diff
@@ -1,10 +1,10 @@
 {
-    "ComponentTitle": "Barclays Lets Go Forward",
-    "SeriesURL": "barclays-lets-go-forward",
+    "ComponentTitle": "Sweet Veganuary",
+    "SeriesURL": "sweet-veganuary",
     "Brand": "Barclays",
```

Comment:
Minor: this is probably irrelevant since file is for local testing of layouts

Did you want to also change this line to update the brand too?

Reply:
All done (in the other commit)!